tests/FsCodec.NewtonsoftJson.Tests/SomeNullHandlingTests.fs

```diff
@@ -9,10 +9,7 @@ let serdes = Serdes Options.Default
 
 let [<Fact>] ``Options.Create does not roundtrip Some null`` () =
     let value : string option = Some null
-    let ser = serdes.Serialize value
-    "null" =! ser
-    // But it doesn't roundtrip
-    value <>! serdes.Deserialize ser
+    raises <@ serdes.Serialize value @>
```

Comment:
Probably in SerdesTests.fs, would be good to have characterization tests that illustrate the default config for STJ and NSJ is to silently do roundtripping of `null` (and the fact that `Serdes <| Options.CreateDefault()` preserves that), see 
 `type StjCharacterization`

Reply:
Resolving, see comment about illustrating that some properties hold with and without the rejection converter in other places